patches/chrome-browser-resources-local_discovery-local_discovery.js.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/browser/resources/local_discovery/local_discovery.js b/chrome/browser/resources/local_discovery/local_discovery.js
+index 0b9bc336cc97016029b14b9a41511a74009a7732..5efa92737feb0b01083e890630b370b17e03a418 100644
+--- a/chrome/browser/resources/local_discovery/local_discovery.js
++++ b/chrome/browser/resources/local_discovery/local_discovery.js
+@@ -634,15 +634,6 @@ cr.define('local_discovery', function() {
+     $('cloud-devices-retry-link')
+         .addEventListener('click', retryLoadCloudDevices);
+ 
+-    $('cloud-devices-login-link')
+-        .addEventListener('click', cloudDevicesLoginButtonClicked);
+-
+-    $('register-login-link')
+-        .addEventListener('click', registerLoginButtonClicked);
+-
+-    $('register-overlay-login-button')
+-        .addEventListener('click', registerOverlayLoginButtonClicked);
```

Comment:
Would just removing the text in the <a> be enough for them not to show up? Then we could leave the click handlers maybe?

Reply:
If I just remove the links, the page gets broken:

![chrome-devices](https://user-images.githubusercontent.com/984737/46203295-a4b93500-c319-11e8-933d-9634ade72ab2.png)